server/Game.ts

```diff
@@ -328,6 +369,8 @@ export class Game {
           };
         })
         .sort((pr1, pr2) => pr2.score - pr1.score);
+
+      this.startKickAllPlayersCountdown();
```

Comment:
i would rather start it in the place that sets game stage to GAME_OVER than in updateGameInfo method, that one can be called multiple times